colors/one-nvim.vim

```diff
@@ -194,6 +195,7 @@ local highlight_groups = {
      Number         = { fg = hue_6, bg = none},
      Boolean        = { fg = hue_6, bg = none},
      Float          = { fg = hue_6, bg = none},
+     FloatBorder    = { fg = mono_1, bg = none},
```

Comment:
I think this should be moved up to the "Syntax Groups" section. It's not related to the `Float` numeric type

Reply:
thx! fixed!